lib/list.js

```diff
@@ -0,0 +1,17 @@
+const wreck = require('wreck');
+const Boom = require('boom');
+
+module.exports = async function(email) {
+  const config = this;
+
+  if (email === 'undefined') {
+    return Boom.notFound();
+  }
+
+  const response = await wreck.get(`${config.host}/api/users/list?email=${email}`, { json: true });
```

Comment:
I believe the list endpoint also has pagination too - so we should have the ability to pass that back.